server/graphql/v2/input/OrderUpdateInput.ts

```diff
@@ -29,5 +30,9 @@ export const OrderUpdateInput = new GraphQLInputObjectType({
       type: GraphQLFloat,
       description: 'Host fee percent to be applied to the order',
     },
+    fundReceivedDate: {
```

Comment:
This doesn't follow the pattern we have for dates in the API.

Can we try to get new fields in the API consistent when we introduce them?

Reply:
Thanks. This is now renamed to `processedAt`.